.travis.yml

```diff
@@ -0,0 +1,16 @@
+services:
+- docker
+
+before-script:
+  - docker build -t test .
+
+script:
```

Comment:
You could encrypt the environment variables into this file: https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml